backends/v2v/cfg/convert_destination.cfg

```diff
@@ -32,6 +32,8 @@ variants:
         remote_node_password = RHV_NODE_PASSWORD
         remote_user = ${remote_node_user}
         remote_pwd = ${remote_node_password}
+        ovirt_ca_file_path = "/etc/pki/ovirt-engine/ca.pem"
+        local_ca_file_path = "/tmp/rhv_upload_ca.pem"
```

Comment:
If these paths are not constant values, they should be put as 'XXX_V2V_EXAMPLES' and be replaced when run the cases.

Reply:
They are constant values.